### PR TITLE
[#IO-4] Pipeline apim backup

### PR DESCRIPTION
**Pull request by uolter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # io-infrastructure-pipelines
 
-This repository collects a set of pipelines aming to automate some common and ripetitive task useful to maintein the cloud infrastructure hosting the backend of the [mobile app IO](https://io.italia.it/).
+This repository collects a set of pipelines aming to automate some common and ripetitive tasks useful to maintein the cloud infrastructure hosting the backend of the [mobile app IO](https://io.italia.it/).
```
Comment (balanza) [inline README.md]: typo
```suggestion
This repository collects a set of pipelines aming to automate some common and ripetitive tasks useful to maintain the cloud infrastructure hosting the backend of the [mobile app IO](https://io.italia.it/).
```

File: pipelines/backup-apim-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    default: apimbackup
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+jobs:
+  - job: backup_apim
+    steps:
+      # 1. Run power shell task to backup apim instance into a storage account.
+      - task: AzurePowerShell@5
```
Comment (balanza) [inline pipelines/backup-apim-pipeline.yaml:46]: Why PowerShell and not bash?

Comment (uolter) [reply pipelines/backup-apim-pipeline.yaml:46]: That's what the guys from the support suggested us in yesterday's meeting. It looks like it's more complicated with the az cli.
Last but not least I learned a bit of Power Shell :-) ... First time for me ...

Comment (balanza) [reply pipelines/backup-apim-pipeline.yaml:46]: Idk Powershell good enough to say this is an issue as it is in bash, but be sure that the whole task fails as soon as one of its instructions does (that is, what in bash we prevent by using `set -e` modifier)

Comment (fume) [reply pipelines/backup-apim-pipeline.yaml:46]: hi @balanza @uolter , you can change the FailOnStandardError (default: false) task parameter to accomplish the objective (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops#yaml-snippet)

**Pull request merged**